### PR TITLE
Deployment automation for Cloud Datastream

### PR DESCRIPTION
**Pull request by arunsingh2803**
This python script can be used to deploy cloud datastream streaming data from Cloud SQL (MySQL) to Cloud Storage.
Since terraform do not have a module for cloud datastream yet, this python script can be used as a workaround for automated deployment

### CODE REVIEW

File: tools/datastream-deployment-python-automation/create_connection_profiles_and_datastream.py [hunk 1]
```diff
@@ -0,0 +1,268 @@
+#!/usr/bin/python3
+
+# Copyright 2019 Google LLC
```
Comment (iht) [inline tools/datastream-deployment-python-automation/create_connection_profiles_and_datastream.py:3]: Copyright 2022

File: tools/datastream-deployment-python-automation/variables.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 Google LLC
```
Comment (iht) [inline tools/datastream-deployment-python-automation/variables.py:1]: Copyright 2022

**Review by iht — CHANGES_REQUESTED** [review]
Please fix the copyright year. Other than that, this LGTM :).

Thanks for your contribution to the repo!

File: tools/datastream-deployment-python-automation/create_connection_profiles_and_datastream.py (hunk 1 above)
Comment (arunsingh2803) [reply tools/datastream-deployment-python-automation/create_connection_profiles_and_datastream.py]: Updated, Thanks!

File: tools/datastream-deployment-python-automation/variables.py (hunk 2 above)
Comment (arunsingh2803) [reply tools/datastream-deployment-python-automation/variables.py]: Updated, Thanks!

**Review by iht — APPROVED** [review]
LGTM

**Comment by iht** [comment]
Thanks for your contribution to the repo!

**Pull request merged**